spec/lib/crawler/coordinator_spec.rb

```diff
@@ -761,7 +761,7 @@ def extract_links(crawl_result, crawl_depth: 1)
 
     it 'should extract valid links' do
       links = extract_links(crawl_result)
-      expect(links.count).to eq(3)
+      expect(links.count).to eq(2)
```

Comment:
This number change is intentional; Nokogiri would extract links even if the link content was invalid (e.g. `foo%` instead of `/link/to/somewhere`), while Jsoup discards these links.